docs/glossary.md

```diff
@@ -4,32 +4,37 @@ title: Glossary
 sidebar_label: Glossary
 ---
 
+import Hidden from '@site/src/components/Hidden';
+
 # Monetized web page
```

Comment:
I wonder why this is a large title on the page (as compared to the other ones):
<img width="1089" alt="Screenshot 2023-05-23 at 13 39 52" src="https://github.com/WICG/webmonetization/assets/15069181/b97a16b8-11bd-4ce1-ac4d-9f7785d80069">

Reply:
Maybe we can just convert all of these to use ## (header 2) size, and just add `# Glossary` at the beginning